.github/workflows/build_and_publish.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish to PyPI
+
+on:
+ release:
+  types:
+   - created
+
+jobs:
+ build-n-publish:
+  name: Build and publish Python distribution to PyPI
+  runs-on: ubuntu-latest
+  permissions:
+    id-token: write
+  steps:
+   - name: Check out git repository
+     uses: actions/checkout@v4
+
+   - name: Set up Python 3.8
+     uses: actions/setup-python@v5
+     with:
+      python-version: 3.8
+
+   - name: Install build tools
+     run: >-
+        python -m
+        pip install
+        wheel
+        twine
+        --user
+
+   - name: Build a binary wheel and a source tarball
+     run: >-
+        python
+        setup.py
+        sdist
+        bdist_wheel
+
+   - name: Publish distribution 📦 to PyPI
+     uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
I think this will work quite well as is, based on Stranger and https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/. But if there are any problems with setting a trusted GitHub repo, remember you can back up and use a tokenized version as well. I note I'm not a PyPi admin for this repo, but there are plenty so just ask e.g. @jemten or @torbjorgen to follow the instructions in this link an all should be good.